package.json

```diff
@@ -47,6 +49,8 @@
     "ganache-cli": "6.1.8",
     "mocha": "5.2.0",
     "truffle": "5.0.6",
+    "ts-generator": "0.0.8",
```

Comment:
Do we need this? Isn't this a sub-dependency? Where do we use it?

Reply:
It's used in npm script 
```
    "generate": "ts-generator ts-generator.json",

```

It's recommended in `typechain` readme. Refer [here].(https://www.npmjs.com/package/typechain#ts-generator-unified-config)

It's kind of wrapper generator which supports multiple kinds of typescript generator. We can specify the generator in the config file `ts-generator.json`.   `typechain` works without this, it's a more elegant way.